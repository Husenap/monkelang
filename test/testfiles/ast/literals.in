monke say 1
monke say 2
monke say true
monke say false
