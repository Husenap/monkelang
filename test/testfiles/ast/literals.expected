PrintStmt: 
  NumberExpr: 1
PrintStmt: 
  NumberExpr: 2
PrintStmt: 
  BooleanExpr: true
PrintStmt: 
  BooleanExpr: false
